cryptofeed/exchange/ftx.py

```diff
@@ -44,8 +46,23 @@ def _parse_symbol_data(cls, data: dict, symbol_separator: str) -> Tuple[Dict, Di
             symbol = d['name']
             ret[normalized] = symbol
             info['tick_size'][normalized] = d['priceIncrement']
+            if symbol[-4:] == 'PERP':
```

Comment:
FTX has all kinds of strange instruments and I wasn't entirely sure how to distinguish ones that have an index associated with them. Here I limited the indexes to instruments that have a perpetual future (which limits out duplicates of the same trading pair, and stock-based smart contracts like AAPL). Let me know what you think about this approach.